spark_client/domain.py

```diff
@@ -176,30 +176,35 @@ def __init__(self, environ: Dict = dict(os.environ)):
         self.environ = environ if environ is not None else {}
 
     @property
-    def spark_folder(self) -> str:
+    def spark_home(self) -> str:
+        """Return SPARK_HOME folder"""
+        return self.environ.get("SPARK_HOME", os.path.dirname(__file__))
+
+    @property
+    def spark_install_base(self) -> str:
         """Return the SNAP folder"""
-        return self.environ.get("SNAP", self.environ.get("SPARK_HOME"))
+        return self.environ.get("SPARK_INSTALL_BASE", self.spark_home)
 
     @property
-    def static_conf_file(self) -> str:
-        """Return static config properties file packaged with the client snap."""
-        return f"{self.spark_folder}/conf/spark-defaults.conf"
+    def spark_user_home(self) -> str:
+        """Return SPARK_HOME folder"""
+        return self.environ.get("HOME", self.spark_install_base)
 
     @property
-    def dynamic_conf_file(self) -> str:
-        """Return dynamic config properties file generated during client setup."""
-        return f"{self.environ.get('SNAP_USER_DATA', self.environ.get('HOME'))}/spark-defaults.conf"
```

Comment:
This default value wasn't used so far. Do we need it?

Reply:
Good point! I'm inclined to say yes, but more on a SNAP context (which now it does not exists anymore given we are in a python package that should not be binded by a SNAP). Let's remove that and decide later how to reintroduce this file, but rather in the SNAP